fsw/src/sc_cmds.c

```diff
@@ -102,7 +102,7 @@ void SC_ProcessAtpCmd(void)
                  **
                  */
                 CFE_MSG_ValidateChecksum(&EntryPtr->Msg, &ChecksumValid);
-                if (ChecksumValid == true)
+                if (true) //(ChecksumValid == true)
```

Comment:
This is an odd looking change.

Reply:
This is due to us not calculating Checksums as part of the table generation process in NOS3. Ideally resolved in future issue.